report/3.Theory/SemanticsDescription.tex

```diff
@@ -591,100 +637,78 @@ \subsection{Big-Step Semantic for Variable Declarations}
 \textbf{[$Dcl_VAssExp_{BSS}$]}\\
 \begin{center}
 	\begin{math}
-		{\langle T\ x = e;\ S,\ env_V,\ sto \rangle \rightarrow_{D_V} (env_V'[x \mapsto l][next \mapsto l'],\ sto')}
+		{\langle T\ x = e,\ env_V,\ sto \rangle \rightarrow_{D_V} (env_V'[x \mapsto l][next \mapsto l'],\ sto')}
```

Comment:
Mangler "givet environments"
Og så skal der stå "sto[l -> v]" i stedet for " sto' "